src/lib/Activity.svelte

```diff
@@ -187,12 +213,15 @@
       }
 
       li {
-        display: flex;
-        flex-flow: row;
-        align-items: center;
-        gap: 8px;
-        width: max-content;
-        max-width: 100%;
+        button {
```

Comment:
We could add `cursor: pointer` to this button so it's more obvious it's clickable.